docs/source/construct-yaml.md

```diff
@@ -239,6 +239,14 @@ Only affects `.sh` installers. If `False`, the installer launches
 an interactive wizard guiding the user through the available options. If
 `True`, the installer runs automatically as if `-b` was passed.
 
+### `force_by_default`
+
+Only affects `.sh` installers. If `True`, the installer will remove the
+existing installation without prompting the user. If set, the user can
```

Comment:
> the installer will remove the existing installation without prompting the user.

The installer does not perform any removal operations. It will just clobber over the existing installations. So, the final installation directory contains a mix of new and old installations. See also https://github.com/conda/constructor/issues/679.

Reply:
ah this is a bug. yeah, i thought that `-f` would remove, then overwrite.

Reply:
a bug in my understanding.